driver/config/config.go

```diff
@@ -782,6 +793,7 @@ func (p *Config) SelfServiceCodeStrategy(ctx context.Context) *SelfServiceStrate
 			Config:  config,
 		},
 		PasswordlessEnabled: pp.BoolF(basePath+".passwordless_enabled", false),
+		MFAEnabled:          pp.BoolF(basePath+".mfa_enabled", false),
```

Comment:
Can the code strategy be now enabled for both passwordless AND MFA at the same time?

Reply:
yes, but a session that already used code for passwordless, does not result in an AAL2 session: https://github.com/ory/kratos/blob/f47c55c8142a087cf92f33ea451429e790da1967/selfservice/strategy/code/strategy_login.go#L64-L78

I think we should probably add a check to login flow creation if `aal=aal2`, to prevent someone from creating an MFA flow if the session used code for passwordless. WDYT?

Reply:
I think that the administrator has to decide whether this method can be used for login or for MFA and we should not allow that this method can be used for both

Reply:
But that limits the feature too much, I fear. Code via Email + Code via SMS is a legitimate use case, and should result in an aal2 session, right? If not, I am fine with adding a config validation to make these mutually exclusive.

Reply:
I see, that's a good point. I will take another look at this.

Reply:
Hello @aeneasr and @jonas-jonas, I'm currently building an app with Ory, using the code via email for login.
I'd like to add MFA with code via SMS, but I understand that it's currently not possible.

Do you think it's a legitimate use-case (code via email + code via sms), and are you planning to support it soon ? 
Also, if you plan to keep this limitation, what is the reason behind ? is it a security concern ?